chapter3/tests/t11/php-fpm.conf

```diff
@@ -0,0 +1,38 @@
+[global]
+pid = /run/php-fpm/php-fpm.pid
+
+error_log = /dev/stderr
+log_level = error
+
+daemonize = no
+events.mechanism = epoll
+
+[www]
+user = nginx
+group = nginx
+listen = /run/php-fpm/www.sock
+listen.owner = nginx
+listen.group = nginx
+
+pm = static
+pm.max_children = 1
+
+access.log = /dev/stderr
+access.format = "%R - %u %t \"%m %r%Q%q\" %s %f %{mili}d %{kilo}M %C%%"
+request_terminate_timeout = 20s
```

Comment:
not a good thing as it will kill the worker
see solution around max_execution_time

Reply:
Default `max_execution_time` is 30s, I increased `request_terminate_timeout` to 40s, is it fine?